packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -294,6 +305,7 @@ const styles = {
     `,
     desktopVisualContainer: css`
         display: none;
+        position: relative;
```

Comment:
Does this have any affect on other banners using the moment banner template?

Reply:
No it shouldn't do but that's because `position: relative;` is equivalent to the default positioning of an element if you don't include an offset of any kind. The only reason I've explicitly given this a `position` value is so it can be the 'parent' to an absolute positioned child.